app/assets/javascripts/modules/global-bar.js

```diff
@@ -73,6 +73,17 @@
           GOVUK.analytics.trackEvent('Global bar', action, {nonInteraction: 1});
         }
       }
+
+      function parseCookie(cookie) {
+        var parsedCookie = JSON.parse(cookie)
+
+        // Tests seem to run differently on CI, and require an extra parse
+        if (typeof parsedCookie !== "object") {
+          parsedCookie = JSON.parse(parsedCookie)
+        }
+
+        return parsedCookie
+      }
```

Comment:
Is there a way this function can be included from the helper file rather than duplicated here?

Reply:
Good shout. I wasn't sure on the best way of doing this as it's used by spec files too. 

I've gone with: moved parseCookie function into `libs/GlobalBarHelper.js` which is required by the global bar JS files and spec helper, but I'm not sure if it's good to be requiring a non-spec file in a spec. Any ideas?

Reply:
That sounds good to me 👍